castai/resource_node_template.go

```diff
@@ -353,7 +352,7 @@ func resourceNodeTemplate() *schema.Resource {
 							Type:     schema.TypeString,
 							Default:  TaintEffectNoSchedule,
 							ValidateDiagFunc: validation.ToDiagFunc(
-								validation.StringInSlice([]string{TaintEffectNoSchedule, TaintEffectNoExecute, TaintEffectPreferNoSchedule}, false),
+								validation.StringInSlice([]string{TaintEffectNoSchedule, TaintEffectNoExecute}, false),
```

Comment:
should generate the docs again to update the fields

Reply:
Documentation is regenerated but there are no changes probably because validation information is not used when generating it

Reply:
Will add allowed values to the field description as we do for some other fields